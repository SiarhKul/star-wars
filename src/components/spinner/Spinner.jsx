import React from "react";
import ClipLoader from "react-spinners/ClipLoader";
import { css } from "@emotion/react";

export const Spinner = () => {
	const override = css`
		width: 20px;
		height: 20px;
		border-color: red;
	`;
	return <ClipLoader css={override} />;
};
