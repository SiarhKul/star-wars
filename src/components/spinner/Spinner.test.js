import { shallow } from 'enzyme';
import React from 'react';
import { Spinner } from './Spinner';

describe('Test Spinner', () => {
  it('render snapshot <Spinner/>', () => {
    const component = shallow(<Spinner />);

    expect(component).toMatchSnapshot();
  });
});
