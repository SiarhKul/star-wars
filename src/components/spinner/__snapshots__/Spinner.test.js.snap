// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Spinner render snapshot <Spinner/> 1`] = `
<Loader
  color="#000000"
  css={
    Object {
      "map": "/*# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIlNwaW5uZXIuanN4Il0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiJBQUtxQiIsImZpbGUiOiJTcGlubmVyLmpzeCIsInNvdXJjZXNDb250ZW50IjpbImltcG9ydCBSZWFjdCBmcm9tIFwicmVhY3RcIjtcclxuaW1wb3J0IENsaXBMb2FkZXIgZnJvbSBcInJlYWN0LXNwaW5uZXJzL0NsaXBMb2FkZXJcIjtcclxuaW1wb3J0IHsgY3NzIH0gZnJvbSBcIkBlbW90aW9uL3JlYWN0XCI7XHJcblxyXG5leHBvcnQgY29uc3QgU3Bpbm5lciA9ICgpID0+IHtcclxuXHRjb25zdCBvdmVycmlkZSA9IGNzc2BcclxuXHRcdHdpZHRoOiAyMHB4O1xyXG5cdFx0aGVpZ2h0OiAyMHB4O1xyXG5cdFx0Ym9yZGVyLWNvbG9yOiByZWQ7XHJcblx0YDtcclxuXHRyZXR1cm4gPENsaXBMb2FkZXIgY3NzPXtvdmVycmlkZX0gLz47XHJcbn07XHJcbiJdfQ== */",
      "name": "eapwt9-override",
      "styles": "width:20px;height:20px;border-color:red;label:override;",
      "toString": [Function],
    }
  }
  loading={true}
  size={35}
  speedMultiplier={1}
/>
`;
